packages/systemd/9015-sd-netlink-make-the-default-timeout-configurable-by-.patch

```diff
@@ -0,0 +1,73 @@
+From de82c39174972d8643ceab49956a6ee66add6f62 Mon Sep 17 00:00:00 2001
+From: Yu Watanabe <watanabe.yu+github@gmail.com>
+Date: Sun, 1 Oct 2023 12:04:59 +0900
+Subject: [PATCH] sd-netlink: make the default timeout configurable by
+ environment variable
+
+On normal systems, triggering a timeout should be a bug in code or
+configuration error, so I do not think we should extend the default
+timeout. Also, we should not introduce a 'first class' configuration
+option about that. But, making it configurable may be useful for cases
+such that "an extremely highly utilized system (lots of OOM kills,
+very high CPU utilization, etc)".
+
+Closes #25441.
+---
+ docs/ENVIRONMENT.md                    |  6 +++++-
+ src/libsystemd/sd-netlink/sd-netlink.c | 23 +++++++++++++++++++++--
+ 2 files changed, 26 insertions(+), 3 deletions(-)
+
+diff --git a/docs/ENVIRONMENT.md b/docs/ENVIRONMENT.md
+index 1e7a75a36c..2fca15ffad 100644
+--- a/docs/ENVIRONMENT.md
++++ b/docs/ENVIRONMENT.md
+@@ -108,7 +108,11 @@ All tools:
+   for example in `systemd-nspawn`, will be logged to the audit log, if the
+   kernel supports this.
+ 
+-`systemctl`:
++* `$SYSTEMD_ENABLE_LOG_CONTEXT` — if set, extra fields will always be logged to
++  the journal instead of only when logging in debug mode.
++
++* `$SYSTEMD_NETLINK_DEFAULT_TIMEOUT` — specifies the default timeout of waiting
++  replies for netlink messages from the kernel. Defaults to 25 seconds.
```

Comment:
This part isn't quite right:
* The `$SYSTEMD_ENABLE_LOG_CONTEXT` variable support didn't get backported so you can just omit that
* The `systemctl:` section header should not go away and should continue to start just before the list with `$SYSTEMCTL_FORCE_BUS=1`